cli/cobra.go

```diff
@@ -28,7 +28,7 @@ func SetupRootCommand(rootCmd *cobra.Command) {
 	rootCmd.SetHelpCommand(helpCommand)
 
 	rootCmd.PersistentFlags().BoolP("help", "h", false, "Print usage")
-	rootCmd.PersistentFlags().MarkShorthandDeprecated("help", "please use --help")
+	rootCmd.PersistentFlags().MarkShorthandDeprecated("help", "please use --help") // nolint
```

Comment:
I'm just curious ... what was the lint error we got for this? I prefer the more specific version of nolint that says the lint check to skip.